src/modules/Map/InteractiveMap/components/Legend/components/Line.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const Line = ({ color, size, strokeWidth = 2 }) => (
+  <div className="tf-legend__symbol">
+    <svg
+      viewBox="0 0 16 16"
+      width={size}
+      height={size}
+    >
+      <line x1="0" y1="0" x2="16" y2="16" style={{ stroke: color, 'stroke-width': strokeWidth }} />
+    </svg>
```

Comment:
If you want to keep the strokeWith in the same value whatever the size, you also need to change the value of the viewBox with the size  

```suggestion
    <svg
      viewBox={`0 0 ${size} ${size}`}
      width={size}
      height={size}
    >
      <line x1="0" y1="0" x2={size} y2={size} style={{ stroke: color, 'stroke-width': strokeWidth }} />
    </svg>
```

Reply:
And set a default value in size to 16 line.3